### PR TITLE
Other Strats

### LINKED ISSUE #6: Other Strategies
## Issue
I'm gonna add a few other strategies!!

## Inspiration
https://www.youtube.com/watch?v=mScpHTIi-kM

## Notes
Gonna start with tit-for-tat probably

### PR DESCRIPTION
**Pull request by hunterjsb**
## Changes
- Added some strategies
- Improved post-game lobby
- Bodied the competition

## Issue
closes #6

### CODE REVIEW

File: cmd/singleplayer/main.go [hunk 1]
```diff
@@ -1,33 +1,143 @@
...
+	}
+	sort.Strings(keys)
+
+	fmt.Printf("%s:\n", title)
+	fmt.Println("---------------")
+	for _, key := range keys {
+		average := float64(totals[key]) / float64(counts[key])
+		fmt.Printf("%-15s : %.2f\n", key, average)
+	}
+}
```
Comment (jcpage573) [inline cmd/singleplayer/main.go:115]: Can you make these methods you can call on []Game?

**Review by jcpage573 — COMMENTED** [review]
The prints should be in game and can the averages be methods off of []Game is that possible?

File: cmd/singleplayer/main.go (hunk 1 above)
Comment (hunterjsb) [reply cmd/singleplayer/main.go:115]: Not without moving all of the logic in `main` into `game.go`

Comment (jcpage573) [reply cmd/singleplayer/main.go:115]: Yeah id like it in game

Comment (jcpage573) [reply cmd/singleplayer/main.go:115]: Having totals and ototals as a method off of []Game would be cleaner then we wouldn't need those maps

**Comment by jcpage573** [comment]
Does that look okay to you?

**Comment by hunterjsb** [comment]
@jcpage573 i think it looks fine for now, could be improved but im ready to move on to other features, so if ur good i think we should merge

**Review by jcpage573 — APPROVED** [review]

**Pull request merged**